api/issue.go

```diff
@@ -186,12 +186,17 @@ type UpdateSchemaGhostContext struct {
 
 // PITRContext is the issue create context for performing a PITR in a database.
 type PITRContext struct {
-	DatabaseID        int                    `json:"databaseId"`
+	DatabaseID int `json:"databaseId"`
+
+	// CreateDatabaseCtx will be not nil if user want to restore to a new database.
 	CreateDatabaseCtx *CreateDatabaseContext `json:"createDatabaseContext"`
 
+	// BackupID is not nil if the user just restore a full backup only.
```

Comment:
Please comment that `BackupID` and `PointInTimeTs` is like an on-off switch that only one is nil and another with value.